server/controllers/index.js

```diff
@@ -155,14 +165,14 @@ router.get('/api/game-state', auth, (req, res) => {
         const currentGameState = Room.getById(roomId).gameState.getState(user.username);
         res.status(200).json({ currentGameState });
     } catch (err) { /* Player attempts to vote for a card again or game status is not appropriate */
+        console.log(err);
         res.status(400).json({ message: err.message });
     }
 });
 
 /* Check if in dev mode, and enable end game request */
 if (process.env.NODE_ENV === 'testing') {
     router.post('/api/reset-server', (req, res) => {
-        db.reset();
```

Comment:
How come there's no reset anymore?

Reply:
No need for the database to be reset, in a real scenario it won't be reset after every game has been played

Reply:
In a real scenario the server wouldn't be reset either but I think it still makes it easier to test, but if it works for now we can leave it, and reset if we get into trouble later on

Reply:
I'll bring the implementation of the function back but won't call it just yet